server/src/main/java/com/google/sps/utils/Ad.java

```diff
@@ -0,0 +1,201 @@
+/* 
+ * Description: Creates ad objects 
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */
+package com.google.sps.utils;
+
+import java.lang.StringBuilder;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List; 
+
+public class Ad {
+
+    public String id; 
+    public String advertiser;  
+    public String startDate;
+    public String endDate;
+    public long impressionsMin; 
+    public long impressionsMax; 
+    public boolean ageTargetingEnabled; 
+    public List<String> genderTarget; 
+    public List<String> geoTarget;
+    public long spendMin; 
+    public long spendMax; 
+    public String headline; 
+    public String link;
+    public String content;  
+    public String headlineSentiment; 
+    public String headlineTerms;
+    public String contentSentiment; 
+    public String contentTerms; 
+
+  public Ad() {
+  // Must have a public no-argument constructor for Firesbase
+  }  
+
+  private Ad(AdBuilder builder) {
+    this.id = builder.id;
+    this.advertiser = builder.advertiser; 
+    this.startDate = builder.startDate; 
+    this.endDate = builder.endDate; 
+    this.impressionsMin = builder.impressionsMin;
+    this.impressionsMax = builder.impressionsMax; 
+    this.ageTargetingEnabled = builder.ageTargetingEnabled;
+    this.genderTarget = builder.genderTarget; 
+    this.geoTarget = builder.geoTarget; 
+    this.spendMin = builder.spendMin; 
+    this.spendMax = builder.spendMax; 
+    this.headline = builder.headline; 
+    this.link = builder.link; 
+    this.content = builder.content; 
+    this.headlineSentiment = builder.headlineSentiment;
+    this.headlineTerms = builder.headlineTerms; 
+    this.contentSentiment = builder.contentSentiment; 
+    this.contentTerms = builder.contentTerms; 
+  }
+
+  @Override 
+  public String toString() {
+    String s = new StringBuilder()
+      .append("ID: " + this.id)
```

Comment:
Nit: I think this should be a 4 space indent, not a 2. 


https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent